go.mod

```diff
@@ -319,3 +319,5 @@ require (
 
 // Using custom fork until https://github.com/onflow/flow-go/issues/5338 is resolved
 replace github.com/ipfs/boxo => github.com/onflow/boxo v0.0.0-20240201202436-f2477b92f483
+
+replace github.com/ethereum/go-ethereum v1.13.5 => github.com/onflow/go-ethereum v0.0.2
```

Comment:
This will break the build for anyone importing `flow-go` as a dependency (that didn't copy this replace). Would it make sense to just change all the imports?

Reply:
I tried that, unfortunately, it won't work unless we replace a lot of internal references in the onflow/go-ethereum.

Reply:
Let me try one more time